### ISSUE TITLE
Make a Casbin middleware for go-zero

### ISSUE THREAD

**Issue by hsluoyz** [casbin/casbin#957]
See requirement from: https://github.com/zeromicro/go-zero/issues/1549

Other middlewares can be used as references: https://casbin.org/docs/en/middlewares

**Comment by SiddheshKanawade** [comment]
@hsluoyz I would like to contribute to this issue, since I am a beginner and its my first issue, can you please guide me to the relevant reading material to get better idea on this.

**Comment by crossphoton** [comment]
This issue is complete right?
Haven't gone through the whole PR though.

**Comment by hsluoyz** [comment]
@crossphoton yes, see: https://github.com/zeromicro/zero-contrib/tree/main/auth/casbin

**Comment by BakeTang** [comment]
hello,but i use this cannot make a difference .Everyone  can pass.But i already use this.

**Comment by hsluoyz** [comment]
Add this to docs: https://casbin.org/docs/middlewares

@PokIsemaine
